### DISCUSSION TITLE
Is it possible to create a collection of documents?

### DISCUSSION THREAD

**Discussion by rchrdnsh** [sveltia/sveltia-cms#60] [Q&A] [answered]
I have a new use case which is for a client to upload pdf documents to Sveltia so people can download them from a page on their website...

Is something like that possible now using Sveltia?

Thank you! XD

**Comment by kyoshino** [comment, marked as answer]
You can use the [File widget](https://decapcms.org/docs/widgets/#file) to upload PDF files. It’s similar to the [Image widget](https://decapcms.org/docs/widgets/#image) but any file type can be supported.

**Reply by rchrdnsh** [reply]
oh great, thank you...will try to set that up :-)
